wiki/Tournaments/OWC/3/en.md

```diff
@@ -0,0 +1,370 @@
+
+### Stage Instructions
+
+1. We are looking for 32 teams from 32 countries. If there are less participants than we want, the tournament will be modeled according to the reached number.
+2. In the first stage (Group Stage), the teams will be divided into 8 groups of 4 team in each one.
+3. All the teams from each group will face each other.
+4. Rankings of each group are determined by sorting the statistics of each team in followed priority:
+   - More matches won.
+   - Have higher `{(the number of maps won) - (the number of maps defeated)}`.
+   - More maps won.
+   - Have higher `∑{(total score difference) / (maximum score)}`.
+   - Winner of the rematch.
+5. Following stages are Knock-Out Stages. This means that the winner ascends to the next stage and the losing team gets kicked out of the tournament.
+6. In Group Stage, you need to win 4 maps to win a match. In Round of 16 and Quarter-finals, you need to win 5 maps to win a match. In Semi-finals and Finals, you need to win 6 maps to with a match.
+
+### Match Instructions
+
+1. 15 minutes before the match, the tournament management must contact each captain and confirm the ready to play. The management creates the match and invites the captains.
+2. During the match, there must be 8 players on the server hosted by one of the captains, 4 players from each team.
+3. Captain of the winning team must provide screenshots of all of the map results and the Multiplayer History link. These must be sent to to ![][flag_PL] [fartownik](https://osu.ppy.sh/users/56917) or ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) via forum PM. If any of the captains won't do it, the match will be nullified and a rematch will be needed.
+4. Each round, the staff will announce the list of beatmaps (= mappool), from which the captains will be able to choose. These beatmaps are seperated into 5 brackets: NoMod, Hidden, HardRock, DoubleTime and Tiebreaker.
+5. Alternately each captain selects a map out of the mappool. Captains should agree to each other who will select the first map. If no agreement can't be reached, blue team starts selecting.
+   - The captains can select maps out of the NoMod bracket unlimited.
+   - Selection out of mod-specific brackets is limited. Each captain can only select one map from each mod-restricted bracket during match. So each team can pick 1 HardRock, 1 Hidden and 1 DoubleTime map during match.
+   - In case of tie, the tiebreaker map must be played.
+6. The settings of the multiplayer room will be: osu! standard mode, `Team VS` mode and win condition: score.
+7. You are free to set 'No Video' mod. Also, you are free to use the Funspoiler options.
+8. The country with best total score on the map wins.
+9. The teams can exchange only one player per map during matches.
+
+### Other Rules
```

Comment:
`Other Rules` -> `Tournament Rules`?

Reply:
i would leave it like that, it was the same way in the forum thread